odbc/src/api/panic_safe_exec_tests.rs

```diff
@@ -22,12 +23,12 @@ mod unit {
     }
 
     #[named]
-    fn non_panic_fn(stmt_handle: HStmt) -> SqlReturn {
+    unsafe fn non_panic_fn(stmt_handle: HStmt) -> SqlReturn {
```

Comment:
You can't have an unsafe block inside another one. It is better to have the unsafe block where they are needed, so I decided to move them down instead of just wrapping things blindly in an unsafe block.
But I think both approach would at least compile ;)